bundle/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1653,4 +1653,5 @@ spec:
     name: operator
   - image: ghcr.io/complianceascode/k8scontent:latest
     name: profile
-  version: 1.4.0
+  replaces: compliance-operator.v1.4.0
```

Comment:
The NVR part of this change needs to be incorporated into `Makefile` tooling.

Reply:
@rhmdnd https://github.com/ComplianceAsCode/compliance-operator/pull/524